package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "JS-Restaurant",
+  "version": "1.0.0",
+  "description": "A simple restaurant page built with JavaScript webpack.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

We should have a script for development and production.
```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "webpack --mode development",
    "build": "webpack --mode production"
```